static/js/util/profile_edit.js

```diff
@@ -220,6 +221,57 @@ export function boundCheckbox(keySet: string[], label: string|React$Element<*>):
   );
 }
 
+const onTelChange = R.curry((
+  keySet,
+  countryKeySet,
+  profile,
+  updateProfile,
+  validator,
+  newPhoneNumber,
+  countryObj
+) => {
+  let currentPhoneNumber = _.get(profile, keySet);
+  if ( currentPhoneNumber !== newPhoneNumber ) {
+    let clone = _.cloneDeep(profile);
+    _.set(clone, keySet, newPhoneNumber);
+    _.set(clone, countryKeySet, countryObj.iso2);
```

Comment:
One slight amendment to that previous comment: I suppose the column with change is only really required if there is an output of the tool you're using that exceeds 35 chars.